### ISSUE TITLE
The result is different form paper

### ISSUE THREAD

**Issue by Lukangkang123** [liyunqianggyn/Deep-Unsupervised-Image-Hashing#5]
I tested on the cifar10 dataset, but found that the map was only 0.22
![8481089B53758DE622056967A6C2B83B](https://user-images.githubusercontent.com/38984189/138547789-58adf43c-834f-49c6-bcf6-3bb3235d4ecd.jpg)

**Comment by liyunqianggyn** [comment]
Thanks for your interests! 
Seems the training loss does not converge well at the end. I will run the code to check it and I'll leave another comment here when I have. Please stay tuned! :)

**Comment by liyunqianggyn** [comment]
Hello I run the code with same setting as in released code, 
"train_dataset 50000
test_dataset 10000
database_dataset 50000
Lr = 0.0001
bit = 64
gamma = 6"
**_The mAP can reach 59.6% (in paper it is 59.5%) as far. Here is the training log results_:**
![image](https://user-images.githubusercontent.com/29226200/139406117-95bd9100-67a6-4ebc-9257-1b8449bb0b2a.png)
![image](https://user-images.githubusercontent.com/29226200/139406200-87c057f2-ada5-4c67-b35b-f1a7e3f23cda.png)

**_May I ask which implementation code you are using?_** Yesterday we found a small mistake in another implementation ["DeepHash-pytorch"](https://github.com/swuxyj/DeepHash-pytorch), if you are using it, please refer to the [issue1](https://github.com/swuxyj/DeepHash-pytorch/issues/20) and [issue2](https://github.com/swuxyj/DeepHash-pytorch/issues/18) I've also email this small problem, and the problem has been corrected now.  

Hope it can help you! Please let me know if you have other questions, this is helping us to rethink more of our paper :)

**Comment by Lukangkang123** [comment]
> Hello I run the code with same setting as in released code, "train_dataset 50000 test_dataset 10000 database_dataset 50000 Lr = 0.0001 bit = 64 gamma = 6" **_The mAP can reach 59.6% (in paper it is 59.5%) as far. Here is the training log results_:** ![image](https://user-images.githubusercontent.com/29226200/139406117-95bd9100-67a6-4ebc-9257-1b8449bb0b2a.png) ![image](https://user-images.githubusercontent.com/29226200/139406200-87c057f2-ada5-4c67-b35b-f1a7e3f23cda.png)
> 
> **_May I ask which implementation code you are using?_** Yesterday we found a small mistake in another implementation ["DeepHash-pytorch"](https://github.com/swuxyj/DeepHash-pytorch), if you are using it, please refer to the [issue1](https://github.com/swuxyj/DeepHash-pytorch/issues/20) and [issue2](https://github.com/swuxyj/DeepHash-pytorch/issues/18) I've also email this small problem, and the problem has been corrected now.
> 
> Hope it can help you! Please let me know if you have other questions, this is helping us to rethink more of our paper :)

I downloaded and ran the code again, then got the result of 0.5 +. Thanks you for your reply.
